apps/champions/test/battle_test.exs

```diff
@@ -680,6 +680,158 @@ defmodule Champions.Test.BattleTest do
       assert "team_1" ==
                Champions.Battle.Simulator.run_battle([unit], [target_dummy_5], maximum_steps: maximum_steps).result
     end
+
+    test "Self Heal" do
+      maximum_steps = 5
+
+      # Create a character with a basic skill that will heal 5 damage to itself
+      heal_basic_skill_params =
+        TestUtils.build_skill(%{
+          name: "Heal Self skill",
+          mechanics: [
+            %{
+              trigger_delay: 0,
```

Comment:
```suggestion
```

Reply:
This param is implicit in build_skill

Reply:
Since there isn't a `build_mechanic` util, the `trigger_delay` field must be specified to avoid errors.